DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateProgress/LearningLogEntryViewModelTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Delegates.DelegateProgress
+{
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateProgress;
+    using FizzWare.NBuilder;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class LearningLogEntryViewModelTests
+    {
+        [TestCase(1, "1")]
+        [TestCase(null, "N/A")]
+

```

Comment:
All tests in this file seem to have this extra newline?

Reply:
No idea how I missed that. Should be sorted now.